staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model/registry.go

```diff
@@ -33,7 +33,7 @@ type Resolver interface {
 func NewRegistry(stdExprEnv *cel.Env) *Registry {
 	return &Registry{
 		exprEnvs: map[string]*cel.Env{"": stdExprEnv},
-		schemas: map[string]*schema.Structural{},
```

Comment:
Did you mean to change these other files?  They seem unrelated.

Reply:
They are just some unrelated format refinement. As seans3 explained, I will revert them.

Reply:
Sorry, didn’t see that part of his comment.

Reply:
no problem and reverted :-)